admin/includes/languages/english/orders.php

```diff
@@ -54,38 +52,21 @@
 define('ENTRY_CREDIT_CARD_NUMBER', 'Credit Card Number:');
 define('ENTRY_CREDIT_CARD_CVV', 'Credit Card CVV Number:');
 define('ENTRY_CREDIT_CARD_EXPIRES', 'Credit Card Expires:');
-define('ENTRY_SUB_TOTAL', 'Sub-Total:');
-define('ENTRY_TAX', 'Tax:');
 define('ENTRY_SHIPPING', 'Shipping:');
-define('ENTRY_TOTAL', 'Total:');
 define('ENTRY_DATE_PURCHASED', 'Date Purchased:');
 define('ENTRY_STATUS', 'Status:');
-define('ENTRY_DATE_LAST_UPDATED', 'Date Last Updated:');
 define('ENTRY_NOTIFY_CUSTOMER', 'Notify Customer:');
 define('ENTRY_NOTIFY_COMMENTS', 'Append Comments:');
-define('ENTRY_PRINTABLE', 'Print Invoice');
 
 define('TEXT_INFO_HEADING_DELETE_ORDER', 'Delete Order');
 define('TEXT_INFO_DELETE_INTRO', 'Are you sure you want to delete this order?');
 define('TEXT_INFO_RESTOCK_PRODUCT_QUANTITY', 'Restock product quantity');
 define('TEXT_DATE_ORDER_CREATED', 'Date Created:');
 define('TEXT_DATE_ORDER_LAST_MODIFIED', 'Last Modified:');
 define('TEXT_INFO_PAYMENT_METHOD', 'Payment Method:');
-define('TEXT_PAID', 'Paid');
-define('TEXT_UNPAID', 'Un-paid');
 
 define('TEXT_ALL_ORDERS', 'All Orders');
 
-define('EMAIL_SEPARATOR', '------------------------------------------------------');
-define('EMAIL_TEXT_SUBJECT', 'Order Update');
-define('EMAIL_TEXT_ORDER_NUMBER', 'Order Number:');
-define('EMAIL_TEXT_INVOICE_URL', 'Detailed Invoice:');
-define('EMAIL_TEXT_DATE_ORDERED', 'Date Ordered:');
-define('EMAIL_TEXT_COMMENTS_UPDATE', '<em>The comments for your order are: </em>');
-define('EMAIL_TEXT_STATUS_UPDATED', 'Your order has been updated to the following status:' . "\n");
-define('EMAIL_TEXT_STATUS_LABEL', '<strong>New status:</strong> %s' . "\n\n");
-define('EMAIL_TEXT_STATUS_PLEASE_REPLY', 'Please reply to this email if you have any questions.' . "\n");
```

Comment:
These email texts are used when sending order status updates.

Reply:
Are you sure?  I thought the OSH_* ones were used now.   My  eyes were crossing when I got here so please double check.

Reply:
Oh gosh. Why is OSH re-defining things?  Makes me wanna rip it out.

Reply:
LOL!   It's working well - why change it?

Reply:
If you prefer, I could change OSH to use the OLD constants and get rid of the OSH constants.  Your call.  That might be best as a separate PR though.

Reply:
Let's leave them in for now. 
Cindy's gonna look at combining things.

Reply:
As mentioned in #1342, the OSH_ constants are used by the `zen_update_orders_history` which can (and is in many of my customizations) be used _outside_ of the admin's `orders.php` processing.

Might I suggest that, although the `orders.php` constants 'came first', that the duplications be removed from that script-specific language file and leave the OSH ones in the extra_definitions to stand.

Reply:
Sure but in a separate PR.  Per the title, the scope of this PR is *remove unused defines*.

Reply:
One of the tenets embraced in the implementation of the email-template capability feature was that all the email-related language defines began with `EMAIL_TEXT_`, but the OSH defines break that pattern.

With all the efforts y'all are doing right now to streamline and reduce inconsistencies keeping this one feels odd.

I always hate making life difficult for mod authors, so I'm not wanting to put up a big fight here. Just trying to figure out the best way forward that doesn't confuse end-users any more than necessary, and doesn't require cluttering the docs with "oh, and in this additional special case you'll find a broken pattern here too". :)

Thoughts?

Reply:
Why don't I restore these, and we'll deal with them later.  Plenty of other issues, and this is a big change hitting a lot of files - I'd like to get it checked in sooner rather than later.

Reply:
Added back.